source/usage-examples/watch.txt

```diff
@@ -134,27 +138,29 @@ pipeline filters out the ``delete`` operation:
 
    Received a change to the collection: ChangeStreamDocument{
      operationType=OperationType{value='insert'},
-     resumeToken={"_data": "825EC..."},
+     resumeToken={"_data": "825E..."},
```

Comment:
Used the first 4 characters instead of 5 to conform with the rest of the abbreviated values.